src/accel/EventAction.hh

```diff
@@ -25,18 +24,16 @@ class EventAction final : public G4UserEventAction
   public:
     //!@{
     //! \name Type aliases
-    using SPParams      = std::shared_ptr<SharedParams>;
     using SPTransporter = std::shared_ptr<detail::LocalTransporter>;
     //!@}
 
   public:
-    EventAction(SPParams params, SPTransporter transport);
+    EventAction(SPTransporter transport);
```

Comment:
Make it explicit now that has only one arg 😉

Reply:
right! :)